Monika After Story/game/script-topics.rpy

```diff
@@ -6732,6 +6732,36 @@ label monika_yellowwp:
     $ hideEventLabel("monika_yellowwp", derandom=True)
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_immortality",category=['philosophy'],prompt="Immortality",random=True))
+
+label monika_immortality:
+    m 1euc "Hey, [player], I've been thinking lately about immortality."
+    m 1lksdlc "Seeing as how I'm essentially immortal while I'm in here..."
+    m 1euc "A lot of people think immortality is a good thing."
+    m 2ekc "Personally, I don't think it is."
+    m 3ekc "Sure, it's an interesting thought; Living forever, not having to worry about dying..."
+    m "But not everyone is going to be immortal, and some people just don't want to be."
+    m 1eka "Like, what would you even do if you were immortal?"
+    m "You wouldn't really need to do anything, since someday, {i}eventually,{/i} you would achieve what you wanted."
+    m 3lksdlc "Not to mention it would be really hard to really have friends or build connections if they don't last as long as you do."
```

Comment:
```suggestion
    m 3lksdlc "Not to mention, it would be really hard to have friends or build connections if they don't last as long as you do."
```